client/src/components/HeroCTA.test.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Enzyme, { shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+import HeroCTA from './HeroCTA';
+
+Enzyme.configure({ adapter: new Adapter() });
```

Comment:
Test setup code like this should probably live in src/setupTests.js (see info [here](https://facebook.github.io/create-react-app/docs/running-tests#src-setuptestsjs))

Reply:
I agree here. They need to be in a test folder. I'm used to the __tests__ convention for a folder name where they live but open to whatever works for this project.

Reply:
That's what I normally do but the Jest tutorials that I read had the component test files live right next to the component files. I'll add a tests folder.